flytekit/core/base_task.py

```diff
@@ -285,6 +285,12 @@ def get_k8s_pod(self, settings: SerializationSettings) -> _task_model.K8sPod:
         """
         return None
 
+    def get_sql(self, settings: SerializationSettings) -> _task_model.Sql:
```

Comment:
```suggestion
    def get_sql(self, settings: SerializationSettings) -> typing.Optional[_task_model.Sql]:
```
sorry i missed this.